docs/TBG_macros.cfg

```diff
@@ -134,8 +134,9 @@ TBG_<species>_pos_dbg="--<species>_position.period 1"
 
 
 # Create a particle-energy histogram [in keV] per species for every .period steps
-TBG_<species>_histogram="--<species>_energyHistogram.period 500 --<species>_energyHistogram.binCount 1024    \
```

Comment:
I think you will now need to update the following example's `.cfg` files:

- [ ] FoilLCT
- [ ] WarmCopper
- [ ] Bremsstrahlung
- [ ] WeibelTransverse
- [ ] Bunch
- [ ] Empty
- [ ] KelvinHelmholtz
- [ ] LaserWakefield

(from `grep -iR "energyHistogram" share/picongpu/examples/`)

Reply:
updated in the last commit